src/routes/all-bps/AllBps.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Redux } from 'redux-render'
+import PropTypes from 'prop-types'
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles'
+import { translate } from 'react-i18next'
+// import BPCard from 'components/bp-card/bp-card'
```

Comment:
Remove this comment

Reply:
Removed the entire file, we weren't using it anyways.